src/main/resources/migration/common/V52__terroism_excluded_offences_update.sql

```diff
@@ -0,0 +1,43 @@
+DELETE FROM offence_schedule_mapping WHERE schedule_part_id = (
+SELECT id FROM schedule_part WHERE schedule_id = ( SELECT id FROM schedule WHERE code = 'TEO'));
+
+-- Dropping the constraints for the integration tests, are re-enabled after the inserts (this is not necessary for the real environments as all the expected offences exist)
+ALTER TABLE offence_schedule_mapping DROP CONSTRAINT fk_schedule_to_offence_constraint;
+ALTER TABLE offence_schedule_mapping ALTER COLUMN offence_id DROP NOT NULL;
+
+INSERT INTO offence_schedule_mapping (schedule_part_id, legislation_text, offence_id) VALUES ((SELECT id FROM schedule_part WHERE schedule_id =  (SELECT id FROM schedule WHERE code = 'TEO') and part_number = 1),(SELECT acts_and_sections FROM offence WHERE code = 'CT15003'),(SELECT id FROM offence WHERE code = 'CT15003'));
```

Comment:
I think you can combine this into one statement, instead of looking up from the same table twice
e.g. replace
(SELECT acts_and_sections FROM offence WHERE code = 'CT15003'),(SELECT id FROM offence WHERE code = 'CT15003')
with 0
(SELECT acts_and_sections, id FROM offence WHERE code = 'CT15003')